src/containers/ZoomControls.js

```diff
@@ -33,13 +33,20 @@ const mapDispatchToProps = { updateViewport: actions.updateViewport };
  * ListItem: {paddingBottom: number, paddingTop: number}}}
  */
 const styles = theme => ({
+  divider: {
+    borderRight: '1px solid #808080',
+    display: 'inline-block',
+    height: '24px',
+    margin: '12px 6px',
```

Comment:
Wondering if we should be using the theme's spacing here.  It's 8px so it might be

```js
margin: `${theme.spacing.unit * 1.5}px ${theme.spacing.unit}px`
```

Which would be equivalent to 

```css
margin: '12px 8px'
```